inspire/base/recordext/fields/inspire.cfg

```diff
@@ -519,6 +519,8 @@ title_translation:
 
 
 url:
+    schema:
+        {'url': {'type': 'list', 'force': True}}
```

Comment:
Yeah, it's covered [here](https://github.com/inspirehep/inspire-next/pull/193/files#diff-3b59804856efabb4ecf3b800ac348ee9R464). ;)